### PR TITLE
Update to use standard

### PR DESCRIPTION
**Pull request by musgravejw**
Reference:  https://github.com/onmodulus/internal/issues/208

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,12 +31,12 @@
...
     "pretest": "npm run lint",
-    "lint": "mod-standard",
+    "lint": "standard -v",
     "test": "tape test"
   },
   "dependencies": {
     "@modulus/rabbit-channel": "1.0.0",
-    "serialize-error": "2.0.0"
+    "serialize-error": "2.0.0",
+    "standard": "^8.5.0"
```
Comment (jackboberg) [inline package.json]: move to `devDependencies`

**Review by jackboberg — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -31,13 +38,8 @@
...
     "coverage": "lab -r html -o coverage/index.html -r lcov -o coverage/lcov.info",
-    "pretest": "npm run lint",
-    "lint": "mod-standard",
-    "test": "tape test"
-  },
-  "dependencies": {
-    "@modulus/rabbit-channel": "1.0.0",
-    "serialize-error": "2.0.0"
+    "lint": "standard -v",
+    "test": "standard -v && tape test",
```
Comment (jackboberg) [inline package.json:42]: `npm run lint && npm run unit`

**Review by jackboberg — CHANGES_REQUESTED** [review]

File: package.json [hunk 3]
```diff
@@ -6,13 +6,20 @@
...
   "devDependencies": {
-    "@modulus/standard": "0.6.0",
     "code": "1.5.0",
     "lab": "6.2.0",
     "sinon": "1.17.4",
+    "standard": "8.5.0",
     "tape": "4.6.0"
   },
+  "engines": {
+    "node": ">=6"
```
Comment (theworkflow) [inline package.json:21]: This code is compatible with `node@v0.10.x` and above.

**Review by theworkflow — CHANGES_REQUESTED** [review]

**Comment by jackboberg** [comment]
duplicate #10

**Pull request closed without merging**